reference/configuration/framework.rst

```diff
@@ -71,14 +115,65 @@ http_method_override
 .. versionadded:: 2.3
     The ``http_method_override`` option was introduced in Symfony 2.3.
 
-**type**: ``Boolean`` **default**: ``true``
+**type**: ``boolean`` **default**: ``true``
 
 This determines whether the ``_method`` request parameter is used as the intended
 HTTP method on POST requests. If enabled, the
 :method:`Request::enableHttpMethodParameterOverride <Symfony\\Component\\HttpFoundation\\Request::enableHttpMethodParameterOverride>`
-gets called automatically. It becomes the service container parameter named
-``kernel.http_method_override``. For more information, see
-:doc:`/cookbook/routing/method_parameters`.
+gets called automatically. It becomes the service container
+parameter named ``kernel.http_method_override``.
+
+.. seealso::
+
+    For more information, see :doc:`/cookbook/routing/method_parameters`.
+
+.. _reference-framework-trusted-proxies:
+
+trusted_proxies
+~~~~~~~~~~~~~~~
+
+**type**: ``array``
+
+Configures the IP addresses that should be trusted as proxies. It becomes the
+service container parameter called ``kernel.trusted_proxies``.
+
+.. seealso::
+
+    For more details, see :doc:`/cookbook/request/load_balancer_reverse_proxy`.
+
+.. versionadded:: 2.3
+    CIDR notation support was introduced in Symfony 2.3, so you can whitelist whole
```

Comment:
can we add a lik to http://de.wikipedia.org/wiki/Classless_Inter-Domain_Routing here?